Methane_MarkDown.Rmd

```diff
@@ -111,3 +109,46 @@ ggplot(data= results, aes(x= year, y= value, color= units))+
          color = "Units") + facet_wrap(~variable)
 ```
 ... to show our data side-by-side.
+
+## Sensitivity Analysis
+
+To examine SSP245's Natural methane  sensitivity we first ask hector to fetch the data. We then create a function to run our parameter set to a particular value. In this case we set our value at `358.05`- twenty percent higher than the original value.
+
+```{r run-with-parameters}
+core <- newcore(ini_file)
+run(core)
+paramr<- fetchvars(core, NA, NATURAL_CH4())
+param<- function(core, parameter, value) {
+  old_value <- fetchvars(core, NA, parameter)
+  unit <- as.character(old_value[["units"]])
+  setvar(core, NA, parameter, value, unit)
+  reset(core)
+  run(core)
+  result <- fetchvars(core, 2000:2100)
+  result[["parameter_value"]] <- value
+  result
+}
+```
+
+Then we tell hector to run a range of values. In this case we ask for 20% above and bellow the set value of `NATURAL_CH4`(341Tg) at intervals of 5. 
+
+```{r sen-param}
+run_with_param_range <- function(core, parameter, values) {
+  mapped <- Map(function(x) param(core, parameter, x), values)
+  Reduce(rbind, mapped)
+}
+sensitivity_emission <- run_with_param_range(core, NATURAL_CH4(), seq(272.8, 358.05, 5))
```

Comment:
Clearer and better, I think, would be to

1. `fetchvars` the default value
2. Compute ±10% (or 20% or whatever)
3. Use those values in this `seq` call

Reply:
What line would I add the change on? Also- it's Saturday thanks for the help I know it's not ideal! (Don't feel pressured to respond though if that was the extent on what you wanted to do today)

Reply:
Also the graph you mention- it shows on my mark down, where you commenting to move it higher up?

Reply:
<img width="806" alt="image" src="https://user-images.githubusercontent.com/106257875/174457212-7dbfde3f-d4ee-49cf-8f68-312a5bc713b8.png">

Reply:
The graph ^